src/SwiftUI/SwiftUI/View.cs

```diff
@@ -47,6 +47,16 @@ public unsafe abstract class View : SwiftStruct
 		GCHandle gch;
 		long refCount = 0; // number of refs passed to native code
 
+		protected View()
+		{
+		}
+
+		public View(TaggedPointer data, SwiftType viewType)
+		{
+			this.data = data;
+			this.viewType = viewType;
+		}
+
```

Comment:
Mono coding convention
```suggestion
		protected View ()
		{
		}

		protected View (TaggedPointer data, SwiftType viewType)
		{
			this.data = data;
			this.viewType = viewType;
		}

```

Reply:
Done!